packages/viewer/src/components/scene-tree-table-cell/scene-tree-table-cell.tsx

```diff
@@ -94,6 +94,14 @@ export class SceneTreeTableCell {
   @Prop()
   public recurseParentSelectionDisabled = false;
 
+  /**
+   * A flag that disables selection if the Alt (or Option on Mac) key is held
+   * down. When enabled, selection will occur as though no modifier keys are
+   * pressed.
+   */
+  @Prop()
+  public disableAltKeySelection = false;
```

Comment:
This seems like a very Connect specific flag. Can we generalize selection behavior somehow, so it applies to applications more broadly?